src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -884,5 +894,33 @@ internal void MessageInlineLink_Click(object sender, RoutedEventArgs e)
                 SdkUIUtilities.OpenExternalUrl(uriString);
             }
         }
+
+        // Generates an unique hash value using the properties affect content of error list item.
+        // If any of these properties changes, needs to refresh error list item.
+        internal int GetIdentity()
```

Comment:
>GetIdentity [](http://example.com/codeflow?start=21&length=11)

was thinking about adding PropertyChanged event for each property matters.
But we usually update multiple properties at once and don't want to update the ErrorList multiple times.
Let me know if any thought

Reply:
Please! I had to uninstall sarifer because of all of the error list "updates" (no actually changes, just every time sarifer completed a scan...error list popped up).

Reply:
The Sarifer scans the content of current editor window after it opens, and re-scan after any keyboard input. It removes all existing results and adds the "new" results back, even the results are same. It should be a good place to call "update" instead of removing all and adding back.

You can disable the scan in Sarifer option to avoid uninstallation for now. :grinning: